public/components/health-check/services/check-api.service.ts

```diff
@@ -33,11 +33,24 @@ const trySetDefault = async (checkLogger: CheckLogger) => {
             return hosts[i].id;
           }
         } catch (err) {
-          checkLogger.info(`Could not connect to API id [${hosts[i].id}]: ${err.message || err}`);
-          errors.push(`Could not connect to API id [${hosts[i].id}]: ${err.message || err}`);
+          checkLogger.info(
+            `Could not connect to API id [${hosts[i].id}]: ${
+              err.message || err
+            }`,
+          );
+          errors.push(
+            `Could not connect to API id [${hosts[i].id}]: ${
+              err.message || err
+            }`,
+          );
         }
       }
       if (errors.length) {
+        if (errors[0].includes('ERROR3099 - 405: Method Not Allowed')) {
+          return Promise.reject(
+            'No API available to connect. This may be related to a version mismatch between Wazuh Server and Wazuh Kibana plugin. Please check the versions and try again.',
```

Comment:
**nitpick**: We should avoid using hardcoded values for the platform name.

Reply:
Resolved in https://github.com/wazuh/wazuh-kibana-app/pull/4964/commits/dc4300c581cdccb125bbed6b0922dca7a1a605ee